lib/openshift-rest-client.js

```diff
@@ -98,7 +98,11 @@ function openshiftClient (settings = {}) {
 
   if (client.settings.config !== undefined) {
     const url = client.settings.config.cluster;
-    client.settings.config.cluster = url.endsWith('/') ? url.slice(0, url.length - 1) : url;
+    if (url === undefined) {
```

Comment:
Would it make sense to check for null and empty string as well?
If yes, you can then just use  `if (!url) {`

Reply:
Yes I think I'll update it. Thanks @matskiv